PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -1907,11 +1910,11 @@ DWORD WINAPI getFileAttributesExWorker(void* data)
 	return TRUE;
 };
 
-DWORD getFileAttributesExWaitSec(const wchar_t* filePath, WIN32_FILE_ATTRIBUTE_DATA* fileAttr, DWORD milleSec2wait, bool* isNetWorkProblem)
+DWORD getFileAttributesExWaitSec(const wchar_t* filePath, WIN32_FILE_ATTRIBUTE_DATA* fileAttr, DWORD milliSec2wait, bool* isNetWorkProblem)
 {
 	GetAttrExParamResult data;
 	data._filePath = filePath;
-	data._attributes = {};
+	data._attributes.dwFileAttributes = INVALID_FILE_ATTRIBUTES;
 	data._result = FALSE;
 	data._isNetworkFailure = true;

```

Comment:
The constructor is a good idea!
However, *_result* member is useful, since according it, we judge whether if the function is the successful
For the others, I would keep them for a while, I will do a clean if they are not necessary.

Reply:
@donho 

>However, _result member is useful...

> ...I will do a clean if they are not necessary

I did not mean to remove the mentioned members by me completely, just the double initialization (once in the struct definition, the second time right after its real instantiation).

Reply:
@xomx 
Indeed, my bad!
Thank you for the review anyway!